server/model/message.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from "mongoose";
+
+const MessageSchema = new mongoose.Schema({
+	clientUsername: { type: String, required: true },
+	stylistUsername: { type: String, required: true },
```

Comment:
Is it possible to use `_id` references instead of username, like it is done in `Appointments.js`?

Reply:
Making this an issue. I promise ill work on this later. Focusing on other priorities right now